web/pandas/config.yml

```diff
@@ -4,59 +4,60 @@ main:
   ignore:
   - _templates/layout.html
   - config.yml
-  - blog.html  # blog will be added at a later stage
   - try.md  # the binder page will be added later
   github_repo_url: pandas-dev/pandas
   context_preprocessors:
   - pandas_web.Preprocessors.navbar_add_info
-  # - pandas_web.Preprocessors.blog_add_posts
+  - pandas_web.Preprocessors.blog_add_posts
   - pandas_web.Preprocessors.maintainers_add_info
   - pandas_web.Preprocessors.home_add_releases
   markdown_extensions:
   - toc
   - tables
   - fenced_code
 static:
-  logo: # path to the logo when it's in the repo
+  logo: # /static/img/pandas.svg
   css:
   - /static/css/pandas.css
 navbar:
-  - name: "Install"
-    target: /install.html
+  - name: "About us"
+    target:
+    - name: "About pandas"
+      target: /about/about.html
```

Comment:
Small nit: we could rename about/about.md to about/index.md? (the url would be a bit nicer I think)